tests/tacacs/test_accounting.py

```diff
@@ -39,7 +39,7 @@ def cleanup_tacacs_log(ptfhost, rw_user_client):
     ssh_run_command(rw_user_client, 'sudo truncate -s 0 /var/log/syslog')
 
 
-def wait_for_log(host, log_file, pattern, timeout=20, check_interval=1):
+def wait_for_log(host, log_file, pattern, timeout=30, check_interval=1):
```

Comment:
Do you see some logs indicating this is a "timing issue"?

Reply:
It's very possible, according to the test error log, the test case failed when check local log, but check server-side log passed. which means accounting works well, seems syslog still not save to file.

        # Verify TACACS+ server and syslog have user command record.
        check_tacacs_server_log_exist(ptfhost, tacacs_creds, "grep")
>       check_local_log_exist(duthost, tacacs_creds, "grep")


I also add debug information after when issue happen.

Reply:
I do not understand why 20->30 change could resolve the timing issue you mentioned. If you intention is to fix "syslog still not save to file", let's save syslog to file, not to change timeout value.

Reply:
Will try journalctl which will show message in memory.